### ISSUE TITLE
"Iodide" link in the top corner of notebook must open iodide homepage in new tab

### ISSUE THREAD

**Issue by bcolloran** [iodide-project/iodide#1268]
otherwise, eval state can be lost if a user navigates from an active notebook to a new page.

this is a general principle: pages should always open in a new tab. I thought this was solved (see #884 #885 #994), but apparently not, so we'll need to look into it again

probably a good first issue

**Comment by pradeepgangwar** [comment]
@bcolloran I will fix this. :)

**Comment by wlach** [comment]
I think we might want to do a deeper rethink of this. Clicking on a logo in a navbar is almost universally code for "open the site in the same tab", I don't think it's great to break the assumption.

I'd suggest we either prompt before switching back to the homepage if clicked directly (using something like this: https://stackoverflow.com/a/1119324) or maybe just amalgamating the hamburger menu with the logo somehow (i.e. make them one button that provides a bunch of options). Here's what glitch does:

![image](https://user-images.githubusercontent.com/20569/49821937-b3748d00-fd49-11e8-957a-5bece0332420.png)

**Comment by hamilton** [comment]
Agreed w/ @wlach. In general the principle should be "users shouldn't lose their work" not "pages should open in new tabs". The latter is a workaround to achieve the former. In favor of an "are you sure you want to lose your work?" sort of modal here. Even if we had gdocs-style editing, we'll still want a modal, since the user is losing state (even if the editor state itself is saved).

Amalgamating logo + dropdown could work, too.

**Comment by bcolloran** [comment]
yep big +1 -- "users shouldn't lose their work" not "pages should open in new tabs" is the important thing.

that said, for now I'd be inclined to accept this PR b/c i think that near term, the risk of losing eval state is greater than the downside of breaking this convention, and we can open a new issue to revisit the matter. but i'm not wedded to that perspective if you guys feel very strongly

**Comment by hamilton** [comment]
mine is a weakly-held belief :) opening in new tab is good & we have a PR already that fixes this for now, so might as well merge it and re-approach later. I suspect we'll be redesigning the toolbar at some point before long at any rate.
